components/alerts/alerts.css

```diff
@@ -0,0 +1,62 @@
+:root{
+    --primary-bg-color: #B3E5FC;
+    --secondary-bg-color: #E0E0E0;
+    --danger-bg-color:#FFCDD2;
+    --success-bg-color: #C8E6C9;
+    --warning-bg-color:#FFE0B2;
+}
+
+.alert {
+    width: 100%;
+    max-width: max-content;
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: space-between;
+    flex-wrap: wrap;
```

Comment:
You can use the flex-shorthand property over here flex-flow:row wrap;

Reply:
Replaced the code with shorthand property